NEWS.rst

```diff
@@ -1,17 +1,16 @@
 .. default-role:: code
 
-Unreleased
+0.4.0 (released ???; uses Hy 0.27.*)
 ======================================================
 
-
 New Features
 ------------------------------
 * `do-n` and `list-n` now allow the count to be `Inf`.
-* New reader macro `#/` that wraps `hy.M` for easy imports in macros.
+* New reader macro `/`.
```

Comment:
Very minor nitpick; should we not leave a brief description of what the `/` macro is/was at this point in time?

Reply:
I get the motivation; I just made the change for consistency, since when a new object or macro is added to Hy or Hyrule, my practice has been to just name it, without a description. Which was a practice that made more sense back when we were adding a lot of stuff on each release. I'll add it back if you want it anyway.

Reply:
I think it's useful to have the quick description, so that folks don't need to search the docs if they're curious about new features.

Reply:
Done.